core/templates/dev/head/pages/exploration_editor/statistics_tab/statistics_tab.html

```diff
@@ -23,7 +23,7 @@ <h3>Card statistics</h3>
               </div>
             </div>
           </div>
-          <div class="container">
+          <div class="container" ng-if="isPlaythroughEnabled">
             <issues-directive/>
```

Comment:
Can we write this as `<issues-directive></issues-directive>`? We typically don't use self-closing tags in our HTML.

Reply:
Done.